[2024-06-25T08:42:32.160+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-25T08:42:32.231+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dbt_snowflake_v5.1.dbt_run scheduled__2024-06-24T00:00:00+00:00 [queued]>
[2024-06-25T08:42:32.269+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dbt_snowflake_v5.1.dbt_run scheduled__2024-06-24T00:00:00+00:00 [queued]>
[2024-06-25T08:42:32.270+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-06-25T08:42:32.304+0000] {taskinstance.py:2330} INFO - Executing <Task(DockerOperator): dbt_run> on 2024-06-24 00:00:00+00:00
[2024-06-25T08:42:32.332+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=1061) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-06-25T08:42:32.323+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dbt_snowflake_v5.1', 'dbt_run', 'scheduled__2024-06-24T00:00:00+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpyvdai_fa']
[2024-06-25T08:42:32.333+0000] {standard_task_runner.py:91} INFO - Job 14: Subtask dbt_run
[2024-06-25T08:42:32.333+0000] {standard_task_runner.py:63} INFO - Started process 1063 to run task
[2024-06-25T08:42:32.414+0000] {task_command.py:426} INFO - Running <TaskInstance: dbt_snowflake_v5.1.dbt_run scheduled__2024-06-24T00:00:00+00:00 [running]> on host 40877dbc4b09
[2024-06-25T08:42:32.574+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dbt_snowflake_v5.1' AIRFLOW_CTX_TASK_ID='dbt_run' AIRFLOW_CTX_EXECUTION_DATE='2024-06-24T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-24T00:00:00+00:00'
[2024-06-25T08:42:32.575+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-25T08:42:32.663+0000] {docker.py:366} INFO - Starting docker container from image ghcr.io/dbt-labs/dbt-snowflake:latest
[2024-06-25T08:42:32.686+0000] {docker.py:374} WARNING - Using remote engine or docker-in-docker and mounting temporary volume from host is not supported. Falling back to `mount_tmp_dir=False` mode. You can set `mount_tmp_dir` parameter to False to disable mounting and remove the warning
[2024-06-25T08:42:37.037+0000] {docker.py:436} INFO - [0m08:42:37  Running with dbt=1.8.2
[2024-06-25T08:42:38.113+0000] {docker.py:436} INFO - [0m08:42:38  Registered adapter: snowflake=1.8.3
[2024-06-25T08:42:38.885+0000] {docker.py:436} INFO - [0m08:42:38  Found 5 models, 9 data tests, 2 sources, 561 macros
[2024-06-25T08:42:38.891+0000] {docker.py:436} INFO - [0m08:42:38
[2024-06-25T08:42:40.798+0000] {docker.py:436} INFO - [0m08:42:40  Concurrency: 1 threads (target='dev')
[2024-06-25T08:42:40.800+0000] {docker.py:436} INFO - [0m08:42:40
[2024-06-25T08:42:40.830+0000] {docker.py:436} INFO - [0m08:42:40  1 of 5 START sql view model dbt_schema.stg_tpch_line_items ..................... [RUN]
[2024-06-25T08:42:41.853+0000] {docker.py:436} INFO - [0m08:42:41  1 of 5 OK created sql view model dbt_schema.stg_tpch_line_items ................ [[32mSUCCESS 1[0m in 1.02s]
[2024-06-25T08:42:41.857+0000] {docker.py:436} INFO - [0m08:42:41  2 of 5 START sql view model dbt_schema.stg_tpch_orders ......................... [RUN]
[2024-06-25T08:42:42.688+0000] {docker.py:436} INFO - [0m08:42:42  2 of 5 OK created sql view model dbt_schema.stg_tpch_orders .................... [[32mSUCCESS 1[0m in 0.83s]
[2024-06-25T08:42:42.692+0000] {docker.py:436} INFO - [0m08:42:42  3 of 5 START sql table model dbt_schema.int_order_items ........................ [RUN]
[2024-06-25T08:42:51.967+0000] {docker.py:436} INFO - [0m08:42:51  3 of 5 OK created sql table model dbt_schema.int_order_items ................... [[32mSUCCESS 1[0m in 9.27s]
[2024-06-25T08:42:51.973+0000] {docker.py:436} INFO - [0m08:42:51  4 of 5 START sql table model dbt_schema.int_order_items_summary ................ [RUN]
[2024-06-25T08:42:54.128+0000] {docker.py:436} INFO - [0m08:42:54  4 of 5 OK created sql table model dbt_schema.int_order_items_summary ........... [[32mSUCCESS 1[0m in 2.15s]
[2024-06-25T08:42:54.132+0000] {docker.py:436} INFO - [0m08:42:54  5 of 5 START sql table model dbt_schema.fct_orders ............................. [RUN]
[2024-06-25T08:42:56.298+0000] {docker.py:436} INFO - [0m08:42:56  5 of 5 OK created sql table model dbt_schema.fct_orders ........................ [[32mSUCCESS 1[0m in 2.16s]
[2024-06-25T08:42:56.303+0000] {docker.py:436} INFO - [0m08:42:56
[2024-06-25T08:42:56.304+0000] {docker.py:436} INFO - [0m08:42:56  Finished running 2 view models, 3 table models in 0 hours 0 minutes and 17.41 seconds (17.41s).
[2024-06-25T08:42:56.378+0000] {docker.py:436} INFO - [0m08:42:56
[2024-06-25T08:42:56.379+0000] {docker.py:436} INFO - [0m08:42:56  [32mCompleted successfully[0m
[2024-06-25T08:42:56.379+0000] {docker.py:436} INFO - [0m08:42:56
[2024-06-25T08:42:56.379+0000] {docker.py:436} INFO - [0m08:42:56  Done. PASS=5 WARN=0 ERROR=0 SKIP=0 TOTAL=5
[2024-06-25T08:42:58.729+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-25T08:42:58.777+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dbt_snowflake_v5.1, task_id=dbt_run, run_id=scheduled__2024-06-24T00:00:00+00:00, execution_date=20240624T000000, start_date=20240625T084232, end_date=20240625T084258
[2024-06-25T08:42:58.836+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-25T08:42:58.874+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-06-25T08:42:58.877+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
